### PR TITLE
PT-249 Fix blip dependencies issues

### PR DESCRIPTION
**Pull request by ydrisR**
Let's switch back to GitHub as a dependency source.
Modify the Docker file to simply copy viz and tideline and then execute npm install && buildapp

### CODE REVIEW

**Comment by ydrisR** [comment]
Ready for review and merge.
Once this is merge we should publish version 0.2.2,
Yourloops project is already configured to use Blip 0.2.2. https://github.com/mdblp/yourloops/pull/109

File: .travis.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ node_js:
 cache: yarn
 
 before_install:
-  - if [[ `npm -v` != 4* ]]; then npm install -g npm@4; fi
+  - if [[ `npm -v` != 6* ]]; then npm install -g npm@6; fi
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.9.2
```
Comment (franck-fourel) [inline .travis.yml]:
```suggestion
```

File: .travis.yml [hunk 2]
```diff
@@ -9,12 +9,12 @@ node_js:
...
   - export PATH="$HOME/.yarn/bin:$PATH"
```
Comment (franck-fourel) [inline .travis.yml]:
```suggestion
```

File: .travis.yml [hunk 3]
```diff
@@ -9,12 +9,12 @@ node_js:
 cache: yarn
```
Comment (franck-fourel) [inline .travis.yml]:
```suggestion
cache: npm
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Blip 
 
-# [1.12.5-dblp.0.2.0] - 2019-03-19 
+# [1.12.5-dblp.0.2.2] - 2019-03-19
```
Comment (franck-fourel) [inline CHANGELOG.md]: is it the proposed new version? 1.12.5-dblp.0.2.2 ?

File: Dockerfile [hunk 5]
```diff
@@ -2,25 +2,25 @@ FROM node:6.10.3-alpine
...
+COPY package.json package.json
+RUN npm install npm@6
+
+RUN chown -R node:node .
 
-RUN yarn install && \
-    yarn cache clean
+USER node
+RUN mkdir -p dist node_modules
+COPY ./node_modules/@tidepool ./node_modules
```
Comment (franck-fourel) [inline Dockerfile]: is it still necessary to copy those modules? 
I would think it is managed by the npm install, line 11
Same for next line, line 18

File: package.json [hunk 6]
```diff
@@ -13,15 +13,15 @@
...
+    "server": "npm run build-config && node server",
     "lint": "eslint app test",
     "install-selenium": "./node_modules/selenium-standalone/bin/selenium-standalone install --version=2.53.0",
     "build-docs": "./update-gh-pages.sh",
     "serve-docs": "./node_modules/.bin/gitbook serve",
     "update-translations": "i18next 'app/**/*.js' 'node_modules/tideline/{plugins,js}/**/*.js' 'node_modules/@tidepool/viz/{storiesDatatypes,storybookDatatypes,stories,src}/**/*.js' -c i18next-parser.config.json -o ."
   },
   "dependencies": {
-    "@tidepool/viz": "0.1.1",
+    "@tidepool/viz": "mdblp/viz#feature/fix-install",
```
Comment (franck-fourel) [inline package.json]: we may need to create a tag for this branch.

File: CHANGELOG.md (hunk 4 above)
Comment (ydrisR) [reply CHANGELOG.md]: yes

File: Dockerfile (hunk 5 above)
Comment (ydrisR) [reply Dockerfile]: I left this for the future, when (someday) we will retrieve tideline and viz from Nexus, but you're right, let's delete this and re-add later when required.

File: package.json (hunk 6 above)
Comment (ydrisR) [reply package.json]: Correct, I'll change this

**Comment by ydrisR** [comment]
@Franck this PR is likely to fail untill we tag viz 0.1.2

**Comment by franck-fourel** [comment]
Tying with this version even if it looks unstable.

**Pull request merged**